ngx_http_auth_ldap_module.c

```diff
@@ -69,6 +70,10 @@ extern int ldap_init_fd(ber_socket_t fd, int proto, const char *url, LDAP **ld);
 
 #define NGX_HTTP_AUTH_LDAP_MAX_SERVERS_SIZE 7
 
+#define SSL_CERT_VERIFY_OFF    0
+#define SSL_CERT_VERIFY_FULL   1
+#define SSL_CERT_VERIFY_CHAIN  2
+
```

Comment:
These constants are used as possible values for `ngx_http_auth_ldap_server_t.ssl_check_cert` to indicate `off`, `on`/`full` and `chain` verification as described in the PR body.